src/app/components/client/exposure_card/ScanResultCard.tsx

```diff
@@ -249,6 +266,24 @@ export const ScanResultCard = (props: ScanResultCardProps) => {
             <dd className={styles.hideOnMobile}>
               {dateFormatter.format(scanResult.created_at)}
             </dd>
+            {props.enabledFeatureFlags?.includes(
+              "DataBrokerRemovalTimeEstimateLabel",
+            ) &&
+              props.experimentData?.["data-broker-removal-time-estimates"]
+                .enabled && (
+                <>
+                  <dt
+                    className={`${styles.hideOnMobile} ${styles.visuallyHidden}`}
+                  >
+                    {l10n.getString(
+                      "dashboard-exposures-filter-exposure-removal-time-title",
+                    )}
+                  </dt>
+                  <dd className={styles.hideOnMobile}>
+                    {removalEstimateTimeLabel}
```

Comment:
I'm not sure if it's intentional, but at a certain screen width, the table headings disappear, while these values are still listed. It might be fine for the other columns, since they're somewhat self-explanatory, but this one might be a bit confusing - so maybe it should disappear the moment the headings do?

(Or maybe the headings shouldn't disappear in general?)

Reply:
Update on the result of the conversation: We’ll address the layout updates together with the redesign of the exposure cards.